### PR TITLE
SLSP-94

### PR DESCRIPTION
**Pull request by willbryanta**
create: added getLibraryItemById to extract individual library items by their Id to show these when a library index item is clicked

### CODE REVIEW

File: components/LibraryIndex/LibraryIndex.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ function LibraryIndex() {
 	const allLibraryItems = libraryItems.map((libraryItem) => (
 		<ul key={libraryItem._id}>
 			<li>
-				<Link to={`library/${libraryItem._id}`}>Name: {libraryItem.name}</Link>
+				<Link to={`${libraryItem._id}`}>Name: {libraryItem.name}</Link>
```
Comment (EddieC97) [inline components/LibraryIndex/LibraryIndex.jsx:19]: it should be {`/library/${libraryItem._id}`}

**Review by EddieC97 — CHANGES_REQUESTED** [review]
See minor requested changes

File: components/LibraryIndex/LibraryIndex.jsx (hunk 1 above)
Comment (willbryanta) [reply components/LibraryIndex/LibraryIndex.jsx:19]: Yeah I thought so too, but for some reason when I click the link from the index it ends up being:
/library/library

So I deleted one instance.

**Comment by Zarbba** [comment]
Hey not sure if we're closing this PR in favour of your other one?

**Pull request closed without merging**